pkg/cli/cmd/radinit/application_test.go

```diff
@@ -74,5 +74,16 @@ func Test_enterApplicationName(t *testing.T) {
 		require.NoError(t, err)
 		require.Equal(t, "another-name", name)
 	})
+	t.Run("Usr is prompted for application name containing uppercase", func(t *testing.T) {
+		ctrl := gomock.NewController(t)
+		prompter := prompt.NewMockInterface(ctrl)
+		runner := Runner{Prompter: prompter}
+
+		setApplicationNamePrompt(prompter, "another-name")
```

Comment:
What is this `another-name`?

Reply:
it just sets the prompter to return that as entered value in mocks. It does not have a significance for what we are trying to test.